cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -117,7 +113,11 @@ def verify_records_rollups_table(appliance, provider):
 
         return False
 
-    wait_for(verify_records_rollups_table, [appliance, provider], timeout=3600,
+    if provider.one_of(SCVMMProvider):
+        wait_for(verify_vm_uptime, [appliance, provider, vmname], timeout=3610,
+           message='Waiting for VM to be up for at least one hour')
+    else:
+        wait_for(verify_records_rollups_table, [appliance, provider], timeout=3600,
```

Comment:
OPTIONAL: increase delay for both ``wait_for``s so it's not run every second

Reply:
Good idea.